CLAUDE.md

```
@@ -0,0 +1,123 @@
+# Quality
+bun lint           # Run TypeScript check + ESLint
+bun test           # Run Vitest tests
+bun format         # Format code with Prettier
+
+# Single test file
+bun test src/utils/config.test.ts
+```
+
+## Architecture
+
+### Technology Stack
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)